app.go

```diff
@@ -66,8 +66,8 @@ func App() *fiber.App {
 
 	app.Use(cors.New())
 	app.Use(sentryfiber.New(sentryfiber.Options{}))
-	app.Get("/", cache.New(cache.Config{Expiration: 1 * time.Hour, CacheControl: true}), r.HomePage)
-	app.Get("/:id", cache.New(cache.Config{Expiration: 1 * time.Hour, CacheControl: true}), r.Get)
+	app.Get("/", cache.New(cache.Config{Expiration: 1 * time.Second, CacheControl: true}), r.HomePage)
+	app.Get("/:id", cache.New(cache.Config{Expiration: 1 * time.Second, CacheControl: true}), r.Get)
```

Comment:
Will change this from `time.Second` to `time.Hour` once the app is stable enough